src/pages/Settings/ExcludedKeywords.js

```diff
@@ -0,0 +1,78 @@
+import { useEffect } from "react";
+import useUnwantedContentPreferences from "../../hooks/useUnwantedContentPreferences";
+import useFormField from "../../hooks/useFormField";
+
+import PreferencesPage from "../_pages_shared/PreferencesPage";
+import Header from "../_pages_shared/Header";
+import { Heading } from "../_pages_shared/Heading.sc";
```

Comment:
should we try to add a rule that if out of a file you're exporting only one component, as it seems to be the case here, you can export it with `default` so it's easier to import?

Reply:
So instead of `export {Heading}`, we would have `export default {Heading}`?
I can change it. Should I modify it for all the styled components used for `pages_shared` and `modal_shared` in this PR?

Reply:
Why not. That way you, if you `export default function Heading` then you can `import Heading from ...` w/o the destructuring operator.

Reply:
and the "why not" is for you can already change things in this PR.

Reply:
✅ Done